generated/gtkd/glib/Timeout.d

```diff
@@ -247,7 +269,7 @@ public class Timeout
 	 * context. You can do these steps manually if you need greater control or to
 	 * use a custom main context.
 	 *
-	 * The interval given is in terms of monotonic time, not wall clock time.
+	 * The interval given in terms of monotonic time, not wall clock time.
```

Comment:
My gir files would be older or newer, should I fix by hand?

Reply:
No, it's only a small documentation change so it's ok.